test_common.go

```diff
@@ -29,6 +29,13 @@ var (
   "y": "4Etl6SRW2YiLUrN5vfvVHuhp7x8PxltmWWlbbM4IFyM",
   "d": "870MB6gfuTJ4HtUnUvYMyJpr5eUZNP4Bk43bVdj3eAE"
 }`
+	ECKey1 = `{
```

Comment:
Can we give this a more meaningful name?  Maybe in concert with the other one.

Also, please make the indentation style similar to the key that precedes it.

Reply:
Done the indentation bit and suggested a name coherent with other names in the file and purpose in the test!